browser/mojSortableTable.ts

```diff
@@ -239,6 +239,53 @@ class PersistentSortOrder {
   }
 }
 
+class MOJFrontendTableSecondOrderSorter {
+  private secondOrderColumnNumber: number | undefined
+
+  constructor(private table: HTMLTableElement) {
+    // 'second-order-column' attribute set in server/utils/viewUtils.ts
+    const tableSecondOrderColumnAttr = this.table.getAttribute('second-order-column')
+    if (tableSecondOrderColumnAttr !== null) {
+      const parsedValue = parseInt(tableSecondOrderColumnAttr, 10)
```

Comment:
Do you think we could call this something like `parsedColumnNumber`? Took a bit of time to figure out what this meant at first as I wasn't sure what `tableSecondOrderColumnAttr` was without checking `viewUtils`.